wazo_sysconfd/modules/resolvconf.py

```diff
@@ -32,6 +29,14 @@
 }
 
 
+class UnsupportedMediaException(Exception):
```

Comment:
I did that because I wanted to follow the same schema as we have here [auth-exceptions](https://github.com/wazo-platform/wazo-auth/blob/master/wazo_auth/exceptions.py) where for each case (error code), we have a different exception, I think it is better. For now I have put it back to use `HttpReqError` like you suggested :)

Reply:
Thank you. Yes, the main idea of this "migration" is to only switch to FastAPI/Python3 only, not to change the way the exceptions are written/managed :-) 

Of course we should write exceptions the same way in all Wazo components, but this should be done later in dedicated PRs :-)